dataScience/src/main/java/edu/csula/datascience/aquisition/deathSource.java

```diff
@@ -59,98 +61,12 @@ public void downloadDeathRecords(String path) throws IOException {
 
 		output.close();
 		inputStream.close();
-		String zipFilePath = "C:\\Users\\Public.DESKTOP-I3193S1\\git\\CS594-Data-Sciences-Spring-2016\\dataScience\\Source.zip";
-		String destDirectory = "C:\\Users\\Public.DESKTOP-I3193S1\\git\\CS594-Data-Sciences-Spring-2016\\dataScience\\Source";
+		String zipFilePath = "Source.zip";
+		String destDirectory = "../DeathSource";

```

Comment:
This is not even implementing the `hasNext()` and `next()` method in any way.

You are just writing your own method to collect data.

In other word, this is not fulfilling the requirement of implementing Collector and Source interface.